### PR TITLE
client: improve multi-client output

### PR DESCRIPTION
**Pull request by robertgzr**

### CODE REVIEW

File: client/bin/multi-client.js [hunk 1]
```diff
@@ -337,18 +337,29 @@ class State {
 					: 1);
 
 			if (yargs.print) {
-				children.forEach(child => {
+				Object.values(children).forEach(child => {
 					console.log(`=====| ${child.outputPath}`);
-					console.log(fs.readFileSync(child.outputPath));
```
Comment (robertgzr) [inline client/bin/multi-client.js:342]: note that I don't know what that is even trying to achieve. in my testing this file never exists...

**Comment by rcooke-warwick** [comment]
@robertgzr do you have an example of what the output looks like with this fix you've made

**Comment by vipulgupta2048** [comment]
Screenshot from 2021-06-09 14-34-27

**Comment by rcooke-warwick** [comment]
@robertgzr @vipulgupta2048 looks good to me

**Review by rcooke-warwick — APPROVED** [review]
LGTM

File: client/bin/multi-client.js [hunk 2]
```diff
@@ -337,18 +337,29 @@ class State {
...
 			}
 		}
 
 		console.log(
-			`Exiting with ${process.exitCode}, input code = ${code}, children: ${map(
+			`Exiting with ${process.exitCode}, client = ${code}, children: ${map(
 				children,
-				c => c.code,
+				c => {
+          switch (c.code) {
```
Comment (vipulgupta2048) [inline client/bin/multi-client.js:356]: I am late, but a default case in this switch here might be great as well. I see some instances where nothing comes back @robertgzr @rcooke-warwick 

```
[79915b6-hup] [Cache used]
[79915b6-hup] Install npm dependencies for suite:
[79915b6-hup] added 59 packages from 68 contributors and audited 59 packages in 11.921s
[79915b6-hup] found 0 vulnerabilities
[79915b6-hup] Removing artifacts from previous tests...
[79915b6-hup] Removing npm dependencies for suite:
[79915b6-hup] (node:246) UnhandledPromiseRejectionWarning: /data/suite/suite.js:56
      const stream = await that.context.get().sdk.getDownloadStream(
                     ^^^^^

SyntaxError: await is only valid in async function
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Suite.resolveTestTree (/usr/app/lib/common/suite.js:181:16)
    at Bluebird.try (/usr/app/lib/common/suite.js:101:25)
(node:246) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:246) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
^CINFO: Cleaning up
INFO: Killing PID 18
Downloading DUT serial log with https://79915b674f93a320470d0850177e384d.balena-devices.com/reports/dut-serial.txt
Downloading artifacts
Exiting with 1, client = 0, children: 
```

Comment (robertgzr) [reply client/bin/multi-client.js:356]: nothing coming back means we ended in a panic like in your example?

Comment (robertgzr) [reply client/bin/multi-client.js:356]: so something like this: 

```patch
diff --git a/client/bin/multi-client.js b/client/bin/multi-client.js
index d857efe..4237929 100755
--- a/client/bin/multi-client.js
+++ b/client/bin/multi-client.js
@@ -358,6 +358,7 @@ class State {
             case 1: return "1 (error)";         // client/lib/index.js#L401
             case 2: return "2 (test failure)";  // client/lib/index.js#L323
             case 3: return "3 (test error)";    // client/lib/index.js#L328
+            default: return "(exception)";
           }
         },
 			).join(',')}`,

```

**Pull request merged**